### PR TITLE
Add awsfargate overview dashboard

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
## What does this PR do?

This PR is to add an overview dashboard for AWS Fargate.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] ~~I have commented my code, particularly in hard-to-understand areas~~
- [x] I have made corresponding changes to the documentation
- [ ] ~~I have made corresponding change to the default configuration files~~
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Related issues

- Relates https://github.com/elastic/beats/issues/22210

## Screenshots
Screen Shot 2020-11-24 at 4 04 57 PM (2)

### CODE REVIEW

File: x-pack/metricbeat/module/awsfargate/cloudformation.yml [hunk 1]
```diff
@@ -63,7 +67,7 @@ Resources:
             - sh
             - -c
           Command:
-            - ./metricbeat setup && ./metricbeat modules disable system && ./metricbeat modules enable awsfargate && ./metricbeat -e -E cloud.id=$ELASTIC_CLOUD_ID -E cloud.auth=$ELASTIC_CLOUD_AUTH
+            - ./metricbeat modules disable system && ./metricbeat modules enable awsfargate && ./metricbeat -e -E cloud.id=$ELASTIC_CLOUD_ID -E cloud.auth=$ELASTIC_CLOUD_AUTH
```
Comment (ChrsMark) [inline x-pack/metricbeat/module/awsfargate/cloudformation.yml:70]: Wondering if we should setup the dashboard by default. I think it is something users would like it, wdyt?

**Review by ChrsMark — COMMENTED** [review]
Looks good! Just 2 comments:

1. I find it difficult to understand the meaning of `Tasks and containers` pie chart. Is there anything missing?
2. Nit: maybe we could also have a screenshot with populated Diskio Visualisation for better user experience.

File: x-pack/metricbeat/module/awsfargate/cloudformation.yml (hunk 1 above)
Comment (kaiyan-sheng) [reply x-pack/metricbeat/module/awsfargate/cloudformation.yml:70]: Yep! I still need to fix this. Plain `./metricbeat setup` doesn't work, I think it also needs `-E cloud.id` and `cloud.auth`. Will update it in next commit 👍

Comment (ChrsMark) [reply x-pack/metricbeat/module/awsfargate/cloudformation.yml]: 👍🏼

**Comment by kaiyan-sheng** [comment]
Need the awsfargate identifier field for dashboard so closing this one and move dashboard over to  https://github.com/elastic/beats/pull/22941

**Pull request closed without merging**